content/docs/user-guide/building/requirements.md

```diff
@@ -0,0 +1,40 @@
+# Requirements
+
+Get `pip` based and `unix` based requirements for your model.
+
+## Description
+
+`mlem` can export the requirements needed for a model using the
```

Comment:
Why is this considered "building" (or exporting) a model if it's not the model you're getting but rather it's dependencies? I'd probably put this in the section [index page](https://mlem-ai-requirement-bui-vdivzc.herokuapp.com/doc/user-guide/building) (which should not be a copy of the [GS page](https://mlem.ai/doc/get-started/building)).